### PR TITLE
More verbose error output: Name errors and blame files

### PR DESCRIPTION
**Pull request by KjellMorgenstern**

### CODE REVIEW

File: bin/fzp/command-validate.go [hunk 1]
```diff
@@ -145,13 +145,16 @@ func validateFile(c *cli.Context, src string) error {
 		// fmt.Println("call validateFile", src)
 		fzpData, _, err := fzp.ReadFzp(src)
 		if err != nil {
-			fmt.Printf("validator failed @ %v\n", err)
+			fmt.Printf("validator failed @ %v, %s\n ", err, src)
```
Comment (ovidiub13) [inline bin/fzp/command-validate.go]: `err` does not need `%v`. You can use directly a string `%s`

Comment (paulvollmer) [reply bin/fzp/command-validate.go]: @ovidiub13 okay, then let us change the `%v` to `%s` at this PR also.

Comment (ovidiub13) [reply bin/fzp/command-validate.go]:
```suggestion
			fmt.Printf("validator failed @ %s, %s\n ", err, src)
```

**Comment by paulvollmer** [comment]
can we delete the `verbose_errors` branch?

**Comment by KjellMorgenstern** [comment]
Thanks for the reminder :-)

**Pull request merged**